src/execution/riscv64/simulator-riscv64.h

```diff
@@ -499,8 +500,10 @@ class Simulator : public SimulatorBase {
   }
   inline int16_t shamt() const { return (imm12() & 0x3F); }
   inline int16_t shamt32() const { return (imm12() & 0x1F); }
```

Comment:
Given the other immediate value getters always specify number of immediate bits, `shamt32` may be misleading (the 32 refers to shifting a 32-bit value, not the number of bits in the shifting amount). I suggest we change the APIs to:
- `shamt` -> `shamt6`
- `shamt32` -> `shamt5`

Thought this is not your code, we may just fix them in one batch